SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialNetDriver.h

```diff
@@ -12,6 +12,7 @@
 #include "EngineClasses/SpatialVirtualWorkerTranslator.h"
 #include "Interop/Connection/ConnectionConfig.h"
 #include "Interop/SpatialOutputDevice.h"
+#include "Interop/SpatialRPCService.h"
```

Comment:
I think you need the header if you are using a uniqueptr right? Just checking. Otherwise we can just declare it at the top.

Reply:
Yes. `TUniquePtr` complains it can't call a destructor on a type that isn't defined if it is just forward declared.

Reply:
Nice!